students/16th/team2/hamin-jo/westagram/user/views.py

```diff
@@ -0,0 +1,75 @@
+import json
+import re
+import bcrypt
+import jwt
+
+from django.views import View
+from django.http import JsonResponse
+
+from user.models import User
+
+class UserView(View):
+    def post(self, request):
+        try:
+            data           = json.loads(request.body)
+            name           = data.get('name')
+            phone          = data.get('phone')
+            password       = data['password']
+            email          = data['email']
+            email_regex    = '^[a-z0-9]+[\._]?[a-z0-9]+[@]\w+[.]\w{2,3}$'
+            password_regex = '[A-Za-z0-9@#$]{8,12}'
+            encoded_pw     = password.encode('utf-8')
+            hashed_pw      = bcrypt.hashpw(encoded_pw, bcrypt.gensalt())
+            decoded_pw     = hashed_pw.decode('utf-8')
+
+            if 'email' in data.keys():
+                if not (re.search(email_regex,email)):
+                    return JsonResponse({'MESSAGE': 'INVALID_EMAIL'}, status=400)
+
+                if User.objects.filter(email=email).exists():
+                    return JsonResponse({'MESSAGE': 'EXIST_USER'}, status=400)
+            else:
+                return JsonResponse({'MESSAGE': 'KEY_ERRORS; email'}, status=400)
+
+            if 'password' in data.keys():
+                if not (re.search(password_regex, password)):
+                    return JsonResponse({'MESSAGE': 'INVALID_PASSWORD'}, status=400)
+            else:
+                return JsonResponse({'MESSAGE': 'KEY_ERRORS; password'}, status=400)
+
+            if phone:
+                if User.objects.filter(phone=phone).exists():
+                    return JsonResponse({'MESSAGE': 'EXIST_USER'}, status=400)
+
+            User.objects.create(name= name, password= decoded_pw, email= email, phone= phone)
```

Comment:
kwargs 부분 `=` 앞뒤로 공백 양쪽 다 추가해주시거나 둘다 없애거나 통일해주세요!
```suggestion
            User.objects.create(name = name, password = decoded_pw, email = email, phone = phone)
```